tests/acceptance/serverd-multi-versions.sh

```diff
@@ -49,7 +49,9 @@ function install(){
     rm -rf "$1"
     if test -z "$2"
     then
-        make install prefix=$PWD/$1/var/cfengine 2>&1 >/dev/null
+        ./configure --enable-debug --with-tokyocabinet --with-init-script --prefix=$PWD/$1/var/cfengine 2>&1 >/dev/null
+        make 2>&1 >/dev/null
+        make install 2>&1 >/dev/null
```

Comment:
This will redirect stderr to stdout, but will not supress it.

Reply:
What do you mean? It's redirected to ``/dev/null``, how more would you want to suppress it?

Reply:
because that's how redirects work - they are evaluated left-to-right, and assign values, not pointers.

So, by default output goes like this:
1=/dev/stdout
2=/dev/stderr
After first assignment (`2>&1`) they are like this:
1=/dev/stdout
2=/dev/stdout
After second assignment (`>/dev/null`) they are like this:
1=/dev/null
2=/dev/stdout

To make it correct, you need to reorder them:
```
>/dev/null 2>&1
```

I've just tested it:
```
lex@flower:~$ cat test.sh
echo err >&2
lex@flower:~$ bash test.sh 2>&1 >/dev/null
err
lex@flower:~$ bash test.sh >/dev/null 2>&1
lex@flower:~$ _
```

Reply:
Wow, I had no idea the order matters like this here! Thanks for the analysis, @Lex-2008! Fixing.

Reply:
Oh, and we had it wrong even before my change.